make/libraries

```diff
@@ -19,17 +19,9 @@ SUNDIALS_NVECSERIAL := $(patsubst %.c,%.o,\
   $(addprefix $(CVODES)/src/, nvec_ser/nvector_serial.c sundials/sundials_math.c))
 
 
-$(sort $(SUNDIALS_CVODES) $(SUNDIALS_NVECSERIAL)) : %.o : %.c
-	@mkdir -p $(dir $@)
-	$(COMPILE.c) -x c -O$(O) -isystem $(CVODES)/include $< -o $@
-
-$(CVODES)/lib/libsundials_cvodes.a: $(SUNDIALS_CVODES)
-	@mkdir -p $(dir $@)
-	$(AR) -rs $@ $^
-
-$(CVODES)/lib/libsundials_nvecserial.a: $(SUNDIALS_NVECSERIAL)
-	@mkdir -p $(dir $@)
-	$(AR) -rs $@ $^
+$(SUNDIALS_CVODES) $(SUNDIALS_NVECSERIAL) : CFLAGS += -isystem $(CVODES)/include -x c
```

Comment:
yeah... you're right regarding the language. GNU make is really a weird tool. I've been using the make manual: https://www.gnu.org/software/make/manual/make.html. It's actually undergone some improvements (I read an older version from cover to cover to figure out most of what's gone into our makefiles).